rmf_site_editor/src/site/deletion.rs

```diff
@@ -114,14 +115,73 @@ impl Plugin for DeletionPlugin {
     }
 }
 
-fn handle_deletion_requests(mut deletions: EventReader<Delete>, mut params: DeletionParams) {
+pub struct DeletionBox {
+    system: BoxedSystem<HashSet<Entity>, HashSet<Entity>>,
+    initialized: bool,
+}
```

Comment:
It would be nice to get rid of the initialized field, a suggestion on the PR comment.